src/Modules/Content/Dnn.PersonaBar.Recyclebin/Components/RecyclebinController.cs

```diff
@@ -332,7 +332,7 @@ public void DeleteUsers(IEnumerable<UserInfo> users)
                 var user in
                     userInfos.Select(u => UserController.GetUserById(u.PortalID, u.UserID))
                         .Where(user => user != null)
-                        .Where(user => user.IsDeleted))
+                        .Where(user => !user.IsDeleted))
```

Comment:
only purge soft deleted user, should not change here

Reply:
Same as previous comment.